docs/code/getting-started.md

```diff
@@ -477,7 +492,7 @@ If you want to get really good really fast, you're going to want to practice. Re
 1. Create a new branch folder "hello-packaging"
 2. Set up our directory structure including the "src" and "data" folders.
 3. Add your Authors, Copying, .desktop, and source code.
```

Comment:
I think now that there is no `AUTHORS` file. And now we have `.appdata.xml` file.
Should be `3. Add your Copying, .desktop, .appdata.xml, and source code.